rosidl_typesupport_introspection_c/resource/msg__introspection_type_support.h.template

```diff
@@ -27,6 +27,8 @@ function_prefix = '%s__%s__rosidl_typesupport_introspection_c' % (spec.base_type
 
 #include "@(spec.base_type.pkg_name)/msg/rosidl_typesupport_introspection_c__visibility_control.h"
 
+@[if not (spec.base_type.type.endswith('Request') or spec.base_type.type.endswith('Response'))]@
```

Comment:
A message can have the same string suffix so this check is too fragile. Maybe check for `_Request`? (Not sure if there is something even better.)